opapi/src/generators/errors.ts

```diff
@@ -39,16 +41,20 @@ abstract class BaseApiError<Code extends ErrorCode, Type extends string, Descrip
   }
 }
 
+function isObject(obj: unknown): obj is object {
```

Comment:
**suggestion:** there's no need to ensure that obj is not an array here
```suggestion
const isObject = (obj: unknown): obj is object => {
  return obj !== null && typeof obj === 'object'
}
```

Reply:
> there's no need to ensure that obj is not an array here

pretty sure it's required
```
> typeof []
'object'
```

Reply:
yes Im aware; it is indeed an object:
```node
> const arr = []
> arr.isApiError === true // false but does not throw

> arr.isApiError = true // it is an object so you can set a property on it
> arr.isApiError === true // true
```

\+ I used an arrow function to make it like the function just below `isApiError`


I should have been more explicit that this comment is **non-blocking**. I tought **suggestion** comments were non-blocking by nature in conventionnal comments, but that's not the case. I already approved the PR, so you can merge if you don't agree with my suggestion